dwave/embedding/chain_strengths.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import math
+
+__all__ = ['scaled_degree_rms']
+
+def scaled_degree_rms(bqm, prefactor=1.414, **kwargs):
```

Comment:
After some thinking about a name for this function, I had in mind that `uniform_torque_compensation` could be a good name for this function. "Torque compensation", because the goal with this method is to compensate the torque that would break the chain, "uniform" because is uniform across all chains. Not a necessary change, but just a naming idea..  @arcondello @randomir @hhtong